ecosystem/sep-0012.md

```diff
@@ -302,6 +340,41 @@ For example:
 }
 ```
 
+## Customer PUT Verification
+
+This endpoint allows servers to accept data values, usually confirmation codes, that verify a previously provided field via `PUT /customer`, such as `mobile_number` or `email_address`. Note that while fields such as `photo_proof_residence` or `notary_approval_of_photo_id` are verifications of other fields described in [SEP-9](#sep-0009.md), the server does not require the associated fields _before_ verification can be accomplished.
```

Comment:
❓ The sentence about photo proof of resident and notary verifications isn't clear to me. What you want to communicate? How do these fields work?

Reply:
`photo_proof_residence` is a verification of the address fields, like `address`, `state_or_province`, etc.

`notary_approval_of_photo_id` is a verification of `photo_id_front` and `photo_id_back`.

However, you wouldn't listed `photo_id_front` as `VERIFICATION_REQUIRED`. Instead you would just also list `notary_approval_of_photo_id` in your field set.

In general, a field only needs `VERIFICATION_REQUIRED` if that field cannot be verified until it is first provided to the server, because if that weren't the case, the server would just request another field that verifies the field needing verification.

Reply:
Another way to think about it is whether or not the server wants to verify a field themselves or _trust a third party_.

Lets say `photo_proof_residence` is required and the client submits a utility bill. The server is trusting the utility company to have verified that the customer lives at the address on the bill. 

`mobile_number` is different because the server doesn't have to trust a third party. Instead it can just verify that the customer has that number themselves using verification codes.